pkg/controller/jobframework/events.go

```diff
@@ -23,4 +23,5 @@ const (
 	ReasonUpdatedWorkload    = "UpdatedWorkload"
 	ReasonFinishedWorkload   = "FinishedWorkload"
 	ReasonErrWorkloadCompose = "ErrWorkloadCompose"
+	ReasonExcessPodDeleted   = "ExcessPodDeleted"
```

Comment:
not quite a jobframework event. Put it in the pod package.

Reply:
right, now moved to the consts section, along with another reason added recently